web/src/features/hfiCalculator/FormatTableAsCSV.ts

```diff
@@ -0,0 +1,253 @@
+            )
+            rowArray.push(station.station_props.fuel_type.abbrev)
+            rowArray.push(!isUndefined(daily) ? daily.status : 'ND')
+            rowArray.push(
+              !isUndefined(daily) && !isUndefined(daily.temperature)
+                ? daily.temperature.toFixed(DECIMAL_PLACES)
+                : 'ND'
+            )
+            rowArray.push(
+              !isUndefined(daily) && !isUndefined(daily.relative_humidity)
+                ? daily.relative_humidity.toFixed(DECIMAL_PLACES)
+                : 'ND'
+            )
+            rowArray.push(
+              !isUndefined(daily) && !isNull(daily.wind_direction)
+                ? daily.wind_direction.toString()
+                : 'ND'
+            )
+            rowArray.push(
+              !isUndefined(daily) && !isUndefined(daily.wind_speed)
+                ? daily.wind_speed.toFixed(DECIMAL_PLACES)
+                : 'ND'
+            )
+            rowArray.push(
+              !isUndefined(daily) && !isUndefined(daily.precipitation)
+                ? daily.precipitation.toFixed(DECIMAL_PLACES)
+                : 'ND'
+            )
+            rowArray.push(grassCureError ? 'ERROR' : printGrassCurePercentage(daily))
+            rowArray.push(!isUndefined(daily) ? daily.ffmc.toString() : 'ND')
+            rowArray.push(!isUndefined(daily) ? daily.dmc.toString() : 'ND')
+            rowArray.push(!isUndefined(daily) ? daily.dc.toString() : 'ND')
+            rowArray.push(!isUndefined(daily) ? daily.isi.toString() : 'ND')
+            rowArray.push(!isUndefined(daily) ? daily.bui.toString() : 'ND')
+            rowArray.push(!isUndefined(daily) ? daily.fwi.toString() : 'ND')
+            rowArray.push(
+              !isUndefined(daily) && !isNull(daily.danger_class)
+                ? daily.danger_class.toString()
+                : 'ND'
+            )
+            rowArray.push(
+              !isUndefined(daily) && !isUndefined(daily.rate_of_spread) && !grassCureError
+                ? daily.rate_of_spread.toFixed(DECIMAL_PLACES)
+                : 'ND'
+            )
+            rowArray.push(
+              !isUndefined(daily) && !isUndefined(daily.hfi) && !grassCureError
+                ? daily.hfi.toFixed(DECIMAL_PLACES)
+                : 'ND'
+            )
+            rowArray.push(
+              !isUndefined(daily) &&
+                !isUndefined(daily.sixty_minute_fire_size) &&
+                !grassCureError
+                ? daily.sixty_minute_fire_size.toFixed(DECIMAL_PLACES)
+                : 'ND'
+            )
+            rowArray.push(!isUndefined(daily) && !grassCureError ? daily.fire_type : 'ND')
+            rowArray.push(
+              !isUndefined(daily) && !grassCureError
+                ? daily.intensity_group.toString()
+                : 'ND'
+            )
+
+            rowsAsStrings.push(CSV.stringify(rowArray))
+          })
+        })
+    })
+    return rowsAsStrings.join('\r\n')
```

Comment:
Does this behave different on windows vs mac? Should it be this?
https://nodejs.org/api/os.html#oseol

Reply:
Deleted `\r\n` because it's no longer needed with the string-to-CSV module we're using now